I'm sorry, but errors were detected in your script. Please correct the
errors listed below, and try again.


File "game/scenes/scene_3.rpy", line 13: expecting name after dot.
    self.xpos.(-4)
              ^

Ren'Py Version: Ren'Py 7.4.8.1895
Thu Oct 21 16:13:56 2021
